connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -399,7 +399,7 @@ message StreamingQueryManagerCommandResult {
   }
 
   message ListStreamingQueryListenerResult {
-    repeated StreamingQueryListenerInstance listeners = 1;
+    repeated string listeners = 1;
```

Comment:
Better to change the name to `listener_ids`. Document the field.

Reply:
Will do, thanks!